requirements.txt

```diff
@@ -1,7 +1,13 @@
 django>=1.8,!=2.1.0,!=2.1.1
 attrs>=17.2.0
-lxml>=3.4
+lxml>=3.4,<4.4 ; python_version < '3.5'
+lxml>=3.4; python_version >= '3.5'
 zeep>=2.4.0
 pycryptodome>=3.7.2
+oscrypto
+asn1crypto
+ocspbuilder
+cryptography>=2.3.1
 
+requests>=2
 wheel==0.32.3
```

Comment:
I think this can be removed from requirements.txt and added to requirements-test.txt as `wheel` (without a version).

Reply:
Not sure if it makes sense in -test, as setup.py reads requirements.txt and where else it's used I don't actually know